vnext/Microsoft.ReactNative/Utils/TransformableText.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <boost/algorithm/string.hpp>
+
+namespace react::uwp {
+struct TransformableText final {
+  enum class TextTransform { Undefined, None, Uppercase, Lowercase, Capitalize };
+
+  TextTransform m_textTransform{TextTransform::Undefined};
+  std::wstring m_originalText{};
```

Comment:
Nit: Do we use m_ for public struct members? Used to seeing capital members without prefixes in other parts of the code.

Reply:
I've seen both (e.g. OInstance uses m_) but I don't feel strongly one way or another